src/miner_consensus_mgr.erl

```diff
@@ -715,7 +715,7 @@ code_change(_OldVsn, State, _Extra) ->
 
 initiate_election(Hash, Height, #state{delay = Delay} = State) ->
     Chain = blockchain_worker:blockchain(),
-    Ledger = blockchain:ledger(Chain),
+    {ok, Ledger} = blockchain:ledger_at(Height + Delay, Chain),
```

Comment:
Do we want to catch or case clause any of these calls to `ledger_at`?

Reply:
I mean, if that fails, does anything that we're doing make any sense?  I guess we could make it so that we don't crash, but honestly I'm not sure I see the value.

Reply:
Yeah, just wanted to ask. Maybe it's better to not crash? We're definitely not gonna progress if `ledger_at` fails though